spec/install/gems/standalone_spec.rb

```diff
@@ -283,9 +283,16 @@
       end
     end
 
+    it "creates stubs that can be symlinked" do
+      require "tmpdir"
+      symlink = File.expand_path("rails", Dir.mktmpdir)
+      File.symlink(bundled_app("bin/rails"), symlink)
```

Comment:
this should fail gracefully if we're on a platform that doesn't support symlink, instead of causing a test failure